elitzur-core/src/main/scala/com/spotify/elitzur/validators/Validator.scala

```diff
@@ -177,6 +177,8 @@ class SeqLikeValidator[T: ClassTag: Validator, C[_]](builderFn: () => mutable.Bu
   }
 
   override def shouldValidate: Boolean = implicitly[Validator[T]].shouldValidate
+  def validationType: String =
```

Comment:
Maybe rename this to `innerValidationType` or something. It should be clear that this is the nested type